public/editor/scripts/editor/js/fc/publisher.js

```diff
@@ -62,11 +62,19 @@ define(function(require) {
       this.updateDialog(publishUrl, true);
     }
 
-    bramble.on("fileChange", publisher.showUnpublishedChangesPrompt.bind(publisher));
-    bramble.on("fileDelete", publisher.showUnpublishedChangesPrompt.bind(publisher));
-    bramble.on("fileRename", publisher.showUnpublishedChangesPrompt.bind(publisher));
-    bramble.on("folderRename", publisher.showUnpublishedChangesPrompt.bind(publisher));
-
+    bramble.on("fileChange", function() {
+      publisher.showUnpublishedChangesPrompt();
```

Comment:
remember to also pass all the arguments down in this rewrite:
```
publisher.showUnpublishedChangesPrompt.apply(publisher, arguments);
```
The old code returned a function that would then be called by `bramble` with any number of arguments, so to preserve that functionality we can't just call the function without any args, whatever bramble passes in needs to be passed on to the handler function